types/auth.go

```diff
@@ -7,5 +7,11 @@ type AuthConfig struct {
 	Auth          string `json:"auth,omitempty"`
 	Email         string `json:"email"`
 	ServerAddress string `json:"serveraddress,omitempty"`
+
+	// IdentityToken is used to authenticate the user and get
+	// an access token for the registry.
+	IdentityToken string `json:"identitytoken,omitempty"`
```

Comment:
Thanks @aaronlehmann for clarifying for me: `Password` is sent with basic auth, but `IdentityToken` goes over `Authorization: Bearer ...`, so the client needs to know which one it is dealing with.